Dockerfile

```diff
@@ -1,6 +1,11 @@
 FROM node:latest
 
-ADD . /app
+RUN mkdir -p /app
 WORKDIR /app
 
+COPY package.json /app/
 RUN npm install
+
+COPY . /app
+
+CMD [ "./zenbot.sh", "trade" ]
```

Comment:
also this command is specified inside docker-compose file so you need to decide which one you want to use to run the command